bindings/kafka/metadata.yaml

```diff
@@ -372,3 +372,10 @@ metadata:
       - "range"
       - "sticky"
       - "roundrobin"
+  - name: headerFromToMetadataExcludedKeysRegex
```

Comment:
this is a big one 😅

NIT: shorten this name. Some suggestions: metaHeadersToIgnoreRegex, excludeMetaFromHeadersRegex, excludeHeaderKeysRegex